tests/notebooks/katib/katib-integration.ipynb

```diff
@@ -26,7 +26,11 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "pytest-skip"
```

Comment:
What is the purpose of this one?

Reply:
The driver skips cells marked with `pytest-skip`, but I still want that cell there for manual execution.